wlauto/common/android/workload.py

```diff
@@ -494,6 +515,22 @@ def reset(self, context):
     def run(self, context):
         ReventWorkload.run(self, context)
 
+        # state detection check if it's enabled in the config
```

Comment:
Since GameWorkload sub classes ReventWorkload it should be a method of ReventWorkload